extras/wait_for_postgres.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+export PGPASSWORD=$POSTGRES_PASSWORD
+RETRIES=10
+
+until psql -h $POSTGRES_HOST -U $POSTGRES_USER -d $POSTGRES_DB -c "select 1" > /dev/null 2>&1 || [ $RETRIES -eq 0 ]; do
+  echo "Waiting for postgres server, $((RETRIES--)) remaining attempts..."
+  sleep 5
+done
```

Comment:
Is this file necessary? I've worked with quite a few docker setups and never encountered anything like that.

Reply:
It's a very common problem that postgres is not ready before django starts attempting to connect.

Reply:
This should be a comment in that file then.